lib/charms/magma_orchestrator_interface/v0/magma_orchestrator_interface.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Library for the magma-orchestrator relation.
+
+This library contains the Requires and Provides classes for handling the magma-orchestrator
+interface.
+
+## Getting Started
+From a charm directory, fetch the library using `charmcraft`:
+
+```shell
+charmcraft fetch-lib charms.magma_orchestrator_interface.v0.magma_orchestrator_interface
+```
+
+Add the following libraries to the charm's `requirements.txt` file:
+- jsonschema
+
+### Requirer charm
+The requirer charm is the charm requiring to connect to an orchestrator from another charm that
+provides this interface.
+
+Example:
+```python
+
+from ops.charm import CharmBase
+from ops.main import main
+
+from lib.charms.magma_orchestrator_interface.v0.magma_orchestrator_interface import (
+    OrchestratorAvailableEvent,
+    OrchestratorRequires,
+)
+
+
+class DummyMagmaOrchestratorRequirerCharm(CharmBase):
+
+    def __init__(self, *args):
+        super().__init__(*args)
+        self.orchestrator_requirer = OrchestratorRequires(self, "orchestrator")
+        self.framework.observe(
+            self.orchestrator_requirer.on.orchestrator_available, self._on_orchestrator_available
+        )
+
+    def _on_orchestrator_available(self, event: OrchestratorAvailableEvent):
+        print(event.root_ca_certificate)
```

Comment:
Are these prints intended here?

Reply:
Yeah, they are just to show the person that will use this lib that those attributes are available to you.

Reply:
Okay :)